### ISSUE TITLE
rxr.open_rasterio(os.path.join(file_dir, filename)).sel(band=1)

### ISSUE THREAD

**Issue by ChangpeiHe** [corteva/rioxarray#689]
RasterioIOError: 'F:/Python/MOLT/MOD11A1.A2020001.h27v05.061.2021003092439.hdf' not recognized as a supported file format.

**Comment by snowman2** [comment]
Unfortunately, there is not enough information here to be able to provide assistance. Please fill out the issue template with the information needed.

**Comment by snowman2** [comment]
Closing due to not enough information provided. It is likely due to your GDAL installation not having the correct drivers installed. 

https://github.com/rasterio/rasterio-wheels/issues/36

I recommend using the GDAL docker images and installing rasterio with the `--no-binary` option or using conda-forge to install rasterio.

**Comment by Berhinj** [comment]
I don't think so, I had to face the same issue today. Install h5py with conda (forge), and you'll see, it'll work!
